kod/object/active/logghost.kod

```diff
@@ -569,6 +574,16 @@ messages:
          piOverlays = piOverlays + 1;
       }
 
+      for overlay in prOverlays
```

Comment:
The prefix "pr" generally refers to a resource name, not a list.  I don't see where prOverlays is declared, kind of confused why this compiles.